kong/plugins/ai-prompt-guard/handler.lua

```diff
@@ -0,0 +1,110 @@
+local _M = {}
+
+-- imports
+local kong_meta = require "kong.meta"
+local buffer    = require("string.buffer")
+--
+
+_M.PRIORITY = 771
+_M.VERSION = kong_meta.version
+
+local function bad_request(msg, reveal_msg_to_client)
+  -- don't let users know 'ai-prompt-guard' is in use
+  kong.log.warn(msg)
```

Comment:
Let's lower the log level for `bad_request`. Consider the audience of `error.log` to be the Kong operator, whose concern is system performance and stability and not necessarily client behavior.

```suggestion
  kong.log.info(msg)
```